src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -42,28 +42,24 @@ public SimpleWrapper(string name, AD7Engine engine, IVariableInformation underly
         public AD7Thread Client { get { return Parent.Client; } }
         public bool Error { get { return Parent.Error; } }
         public uint CountChildren { get { return Parent.CountChildren; } }
-        public bool IsChild { get { return Parent.IsChild; } set {; } }
+        public bool IsChild { get { return Parent.IsChild; } set { ; } }
```

Comment:
The setter is part of the implemented interface. I instead added the expected setter.